src/model.py

```diff
@@ -1985,33 +1985,33 @@ def init_components(self, subtasks):
                     ## CNN component: three possible cnn types for subtasks
                     if expanding_cnn:  # Sometimesse will be 0, other times None
                         components[task][f"cnn_{stack}"] = torch.nn.Sequential(*[
-                            self.expanding_cnn_block(cnn_dim*2, cnn_dim, kernel_size, m=expanding_cnn)
+                            self.expanding_cnn_block((768+cnn_dim), cnn_dim, kernel_size, m=expanding_cnn)
```

Comment:
768 since all tasks get original embeddings 
+
cnn dim, since this will be the output of shared task.